vault/resource_quota_lease_count_test.go

```diff
@@ -14,41 +14,44 @@ import (
 )
 
 func randomQuotaLeaseString() string {
-	whole := acctest.RandIntRange(1000, 2000)
+	whole := acctest.RandIntRange(50000, 60000)
 	return strconv.Itoa(whole + 1000)
 }
 
 func TestQuotaLeaseCount(t *testing.T) {
 	name := acctest.RandomWithPrefix("tf-test")
-	leaseCount := randomQuotaLeaseString()
-	newLeaseCount := randomQuotaLeaseString()
+	ns := "ns-" + name
+	leaseCount := "2"    // randomQuotaLeaseString()
```

Comment:
Are the comments left over and meant to be removed? The way it reads it looks like we got those numbers using the random function?

Reply:
ah good catch, I will restore this

Reply:
So the random bit is no longer necessary since we run the tests in a unique namespace.

Fixed in 99fa2897f8a55c5dbe56989e24965f9c46f19cfe